pack_other.sh

```diff
@@ -78,9 +81,10 @@ if [ -f "gflags_succ" ]; then
 else
 	tar zxf gflags-2.2.0.tar.gz
 	pushd gflags-2.2.0
-	cmake -H. -Bbuild -DCMAKE_INSTALL_PREFIX="$DEPS_PREFIX" -DGFLAGS_NAMESPACE=google -DCMAKE_CXX_FLAGS=-fPIC
-	cmake --build build -- "-j$(nproc)"
-    cmake --build build --target install
+	# Mac will failed in create build/ cuz the dir contains a file named 'BUILD', so we use 'cmake_build' as the build dir
+	cmake -H. -Bcmake_build -DCMAKE_INSTALL_PREFIX="$DEPS_PREFIX" -DGFLAGS_NAMESPACE=google -DCMAKE_CXX_FLAGS=-fPIC
```

Comment:
add `-DBUILD_STATIC_LIBS=ON`

Reply:
when build shared default off, build static will be on